Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN pip3 install --upgrade pip>=10.0.0 &&\
 COPY ./src /src
 
 RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+RUN pip3 install git+https://github.com/humaton/fabric8-analytics-auth.git@devel
```

Comment:
This is installing from your fork's devel branch instead of the latest fabric8-analytics master. Is it intentional?

Reply:
nope that is an error